ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -747,12 +747,12 @@
         'Warehouse A': 'Warenlager A',
         'Warehouse B': 'Warenlager B',
         'Warehouse C': 'Warenlager C',
-        'The wall-mounted right arm begins to move\.\.\.': 'Der wandmontierte rechte Arm ist aktiv!',
-        'The wall-mounted flamethrowers activate\.': 'Die wandmontierten Flammenwerfer sind aktiv!',
-        'The wall-mounted left arm begins to move\.\.\.': 'Der wandmontierte linke Arm ist aktiv!',
-        'You hear frenzied movement from machines beneath\.\.\.': 'Die Maschinenwesen zu deinen Füßen bewegen sich!',
+        'The wall-mounted right arm begins to move\\\\.\\\\.\\\\.': 'Der wandmontierte rechte Arm ist aktiv!',
```

Comment:
Maybe we should just remove the dots on the end of both the original text and the replacement?